File: algorandsamples/mypythondemo/Indexer/BlockInfo.py
import json
# requires Python SDK version 1.3 or higher
from algosdk.v2client import indexer

data = {
    "indexer_token": "",
    "indexer_address": "http://localhost:8980"
}

# instantiate indexer client
myindexer = indexer.IndexerClient(**data)
# gets accounts with a min balance of 100 that have a particular AssetID
data = {
    "block": "555"
}
response = myindexer.block_info(**data)
print(json.dumps(response, indent=2, sort_keys=True))

File: algorandsamples/myjsdemo/Indexer/BlockInfo.js
//BlockInfo.js
const algosdk = require('algosdk');

const indexer_token = "";
const indexer_server = "http://localhost";
const indexer_port = 8980;

// Instantiate the indexer client wrapper
let indexerClient = new algosdk.Indexer(indexer_token, indexer_server, indexer_port);

(async () => {

    let block = 50;
    let blockInfo = await indexerClient.lookupBlock(block).do();
    console.log("Information for Block: " + JSON.stringify(blockInfo, undefined, 2));

    // console.log("Information for Block: " + blockinfo);
})().catch(e => {
    console.log(e);
    console.trace();
});
